packages/block-library/src/list/index.js

```diff
@@ -15,7 +15,7 @@ import settingsV2 from './v2';
 
 const { name } = metadata;
 
-export { metadata, name };
+export { metadata, name, settingsV2 };
```

Comment:
This will be needed just for the mobile release with the feature flag, once the feature flag is removed, this won't be needed.